src/vs/workbench/common/editor/editorGroup.ts

```diff
@@ -211,6 +216,62 @@ export class EditorGroup extends Disposable {
 				}
 			}
 
+			// Insert in Order
```

Comment:
This function is getting too big, I would suggest splitting the sorting into separate functions, the comments will be not needed since the function name will be self-explanatory.